packages/suite/package.json

```diff
@@ -15,7 +15,7 @@
         "translations:list-duplicates": "yarn tsx ../suite-data/src/translations/list-duplicates.ts",
         "translations:list-unused": "yarn tsx ../suite-data/src/translations/list-unused.ts",
         "type-check": "tsc --build tsconfig.json",
-        "test:unit": "jest",
+        "test:unit": "jest --version && jest",
```

Comment:
why add `--version`?

Reply:
because I am interested in this information as long as there are multiple versions

Reply:
why not update all of them in this PR? :D

Reply:
please do it if you have time, will be appreciated